### ISSUE TITLE
Invisible box that prevents clicking

### ISSUE THREAD

**Issue by leynebe** [gnunn1/tilix#1825]
Tilix will sometimes create an invisible box that prevents clicking on other programs and is very faintly visible.

I know it has something to do with Tilix since when I close it, the invisible box is gone and I'm able to use the mouse on that part of my screen in other programs again.

I think it has to do with (me accidentally) alt+left mouse button click dragging a terminal. I say this because the invisible box is exactly the size of the "mini terminal" that is attached to your mouse when you drag it.

I cannot recreate it though and I never use the alt+clickdrag feature intentionally, if there is a way to turn it off for now please let me know.

![systeminfo](https://user-images.githubusercontent.com/34703148/72502624-0e748f80-383a-11ea-8c93-228e9ffd53db.png)

[my tilix dconf](https://github.com/leynebe/tilixconf/blob/master/tilix.dconf)

Great project by the way. Love Tilix!

**Comment by edgan** [comment]
I think you are experiencing issue #1237 .

**Comment by dfaubion-zinc** [comment]
Also experience this (or #1237) on tilix 1.9.3 on xubuntu.

**Comment by acjohnson** [comment]
yeah this is really annoying and took me a long time to figure out it was Tilix doing this. This should be a must fix IMO

**Comment by edgan** [comment]
I am testing this patch. Disabling drag and drop is better than fighting invisible boxes.

**Comment by edgan** [comment]
I am running the patch on two systems without issue. It also seems to have the desired effect, and no unexpected consequences. I can still click on the title bar of the tile to maximize it. Dragging it is disabled as expected.

I may take some time and example the setupDragAndDrop function for ideas of how to fix or workaround this bug without disabling drag and drop.

**Comment by leynebe** [comment]
I haven't had this issue in a long time (it's been 1,5 years since I've created this ticket).
It must've solved itself on version upgrade of tilix or some related package. I didn't post the versions of the terminal then, but currently they are:

```
[ben@work ~]$ tilix -v
Versions
	Tilix version: 1.9.4
	VTE version: 0.64
	GTK Version: 3.24.29

Tilix Special Features
	Notifications enabled=0
	Triggers enabled=0
	Badges enabled=1
```

**Comment by spoonsauce** [comment]
Just had this happen on Fedora 34. Happens pretty regularly if I accidentally drag a session side-bar window.

```
$ tilix -v
Versions
	Tilix version: 1.9.4
	VTE version: 0.64
	GTK Version: 3.24.29

Tilix Special Features
	Notifications enabled=1
	Triggers enabled=0
	Badges enabled=1
```

**Comment by edgan** [comment]
I tried tilix again on Fedora 36, and have had the issue.

**Comment by ronen-fr** [comment]
Fedora 37. 2023. It's still there...

**Comment by leynebe** [comment]
Not encountered it anymore in a long time. Feel free to reopen if you still have the issue.
